experimental/adaptive-tool/README.md

```diff
@@ -1,74 +1,110 @@
-- Different colors and styles for `intent`, `entity`, and `comment` components in LU files
+
+Different colors and styles for `intent`, `entity`, and `comment` components:
 ![lu_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_syntax_highlighting.png)
 
-- Formatting warnings and errors
+Formatting warnings and errors:
 ![lu_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_diagnostic.png)
 
-- Automatic completion of some entities
+Automatic completion of some entities:
 ![lu_completion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_completion.gif)
 
 #### LG
-- Different colors and styles for `template`, `function`, `multi line`, `structure`, `comment`, `condition`, and `switch` components in LG files.
+
+Different colors and styles for `template`, `function`, `multi line`, `structure`, `comment`, `condition`, and `switch` components:
 ![lg_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lg_syntax_highlighting.png)
 
-- Formatting warnings and errors
+Formatting warnings and errors:
 ![lg_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lg_diagnostic.gif)
 
-- Template reference hover
+### Hover, suggestions, and navigation
+
+#### LG
+
+Template reference hover:
 ![template_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/template_hover.png)
 
-- Built-in function hover
+Prebuilt function hover:
 ![function_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/function_hover.png)
 
-- Suggestions for templates and functions
+Suggestions for templates and functions:
 ![function_template_suggestion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/function_template_suggestion.gif)
 
-- Structure property suggestions
+Structure property suggestions:
 ![structure_suggestion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/structure_suggestion.gif)
 
-- Template navigation 
+Template navigation:
+
 ![template_definition](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/template_definition.gif)
 
 ## Debugging
-Adaptive Tool lets developers debug .dialog and LG files in runtime.
+
+Adaptive Tool lets developers debug LG, LU, and Dialog files in runtime. This section covers the steps and shows and example of how to [configure](#configuration), [initialize](#start-bot-runtime-and-complete-initialization), and [debug](#debug-the-runtime) a bot.
```

Comment:
This is one reason I dislike the term _runtime_ for the bot executable. If they are talking about debugging a bot at run time, this edit is okay. If they're talking about debugging the executable at design time or run time, then...ask me and we can figure something out.
```suggestion
Adaptive Tool lets developers debug LG, LU, and dialog files at run time. This section covers the steps and shows an example of how to [configure](#configuration), [initialize](#start-bot-runtime-and-complete-initialization), and [debug](#debug-the-runtime) a bot.
```